tests/gasusage/gasusage_test.go

```diff
@@ -31,9 +32,12 @@ func Test_Staker_GasUsage(t *testing.T) {
 	addReceipt2 := testutil.Send(t, validator2, staker.AddValidator(validator2.Address(), stake, config.MinStakingPeriod))
 	addReceipt3 := testutil.Send(t, validator3, staker.AddValidator(validator3.Address(), stake, config.MinStakingPeriod))
 
-	id1 := addReceipt1.Outputs[0].Events[0].Topics[3]
-	id2 := addReceipt2.Outputs[0].Events[0].Topics[3]
-	id3 := addReceipt3.Outputs[0].Events[0].Topics[3]
+	id1 := addReceipt1.Outputs[0].Events[0].Topics[2]
```

Comment:
May be simpler to do:


```suggestion
	id1 := validator1.Address()
```

Reply:
applied